charts/cloudquery/values.yaml

```diff
@@ -63,6 +63,8 @@ secretRef:
 # -- Schedule fetch time Every 6 hours.
 # More information at: https://crontab.guru/#0_0_*_*_*
 schedule: "0 */6 * * *"
+cronJobLimit: 3
```

Comment:
The presubmit is complaining that the docs aren't autogenerated :) 

Before this variable, add a comment such as:
```
# -- Number of successful cronjobs to retain.
```

Reply:
Also below.

Reply:
[and run `make generate-docs`]

Reply:
I ran the `make generate-docs` and added the description manually, didn't realize there was an actual way to do it (never used helm generate-docs before). Just pushed up a fix, thanks for the `helm-docs` lesson!